tests/test_objects.py

```diff
@@ -573,11 +573,11 @@ def test_merge_no_link_list():
     [
         (
             (
-                {"fruits": ["apple"], "vegetables": ["beet"]},
-                {"fruits": ["banana"], "vegetables": ["carrot"]},
+                {"fruits": ["apple"], "ohters": { "vegetables": ["beet"]}},
```

Comment:
Running `inv black` from the project root or `black tests/test_objects.py` will fix the formatting and make flake8 happy.